hack/test-update-storage-objects.sh

```diff
@@ -123,7 +123,7 @@ startApiServer ${KUBE_OLD_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
 
 
 # Create object(s)
-for test in ${tests[@]}; do
+for test in "${tests[@]}"; do
```

Comment:
change reason:
```
In hack/test-update-storage-objects.sh line 126:
for test in ${tests[@]}; do
            ^-- SC2068: Double quote array expansions to avoid re-splitting elements.
```